src/Compilers/CSharp/Portable/Symbols/Symbol_Attributes.cs

```diff
@@ -410,6 +410,21 @@ internal virtual void PostDecodeWellKnownAttributes(ImmutableArray<CSharpAttribu
                             var boundAttribute = boundAttributeArray[i];
                             Debug.Assert(boundAttribute is not null);
                             NullableWalker.AnalyzeIfNeeded(binders[i], boundAttribute, boundAttribute.Syntax, diagnostics.DiagnosticBag);
+
+                            // Store attribute arguments for local usage analysis.
+                            if (!boundAttribute.ConstructorArguments.IsEmpty || !boundAttribute.NamedArguments.IsEmpty)
```

Comment:
>if (!boundAttribute.ConstructorArguments.IsEmpty || !boundAttribute.NamedArguments.IsEmpty) [](http://example.com/codeflow?start=28&length=91)

I don't think it makes sense to do this analysis for every single attribute that we bind. Attributes outside of a method body cannot refer to a local.